### PR TITLE
Ggpp 80 implement functionality for signup

### PR DESCRIPTION
**Pull request by EricaAnnor**
## Summary
​
> Implement the sign up functionality
​
​
​
## Impact
​
> Web
​
​
​
## Testing
​
> Tested if login card can be displayed and the user can sign up sucessfully. All the tests passed
​
​
​
## Ticket(s) or Conversations
​
>Implement Sign up functionality
​
- https://a2sv.atlassian.net/browse/RE-XXXX
​
## References
​.
​
- ...

### CODE REVIEW

File: web/components/SignupCard.tsx [hunk 1]
```diff
@@ -1,41 +1,98 @@
...
 
 export default function SignupCard() {
-  const [firstName, setFirstName] = useState<string>("");
-  const [lastName, setLastName] = useState<string>("");
-  const [email, setEmail] = useState<string>("");
-  const [password, setPassword] = useState<string>("");
+  // const [firstName, setFirstName] = useState<string>("");
+  // const [lastName, setLastName] = useState<string>("");
+  // const [email, setEmail] = useState<string>("");
+  // const [password, setPassword] = useState<string>("");
```
Comment (tibebugwas) [inline web/components/SignupCard.tsx:11]: remove these commented out lines

File: web/components/SignupCard.tsx [hunk 2]
```diff
@@ -1,41 +1,98 @@
...
+  };
+
+  const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    try {
+      console.log("User:", user);
+      const baseUrl = env("NEXT_PUBLIC_BASE_URL") + "UserAccount/register";
+      const response = await axios.post(baseUrl, user);
+      setMessage("User created successfully!");
+      console.log(response.data);
```
Comment (tibebugwas) [inline web/components/SignupCard.tsx:59]: No need to console.log here

File: web/components/SignupCard.tsx [hunk 3]
```diff
@@ -1,41 +1,98 @@
...
+            className="flex items-center bg-red-100  text-red-700 px-4 py-3 rounded relative"
+            role="alert"
+          >
+            <svg
+              className="fill-current h-6 w-6 text-red-500 mr-4"
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 20 20"
+            >
+              <path d="M10 2C5.58 2 2 5.58 2 10s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8zm1 11H9v-2h2v2zm0-4H9V5h2v4z" />
+            </svg>
```
Comment (tibebugwas) [inline web/components/SignupCard.tsx:90]: You can move this svg into a component and then use the component here. I'm assuming this is the icon but correct me if I'm wrong.

**Review by tibebugwas — CHANGES_REQUESTED** [review]

File: web/components/SignupCard.tsx (hunk 3 above)
Comment (EricaAnnor) [reply web/components/SignupCard.tsx:90]: I was not the one who implemented the sign up card , I just implemented  the functionality

**Review by tibebugwas — APPROVED** [review]

**Pull request merged**